roles/mirror_registry/tasks/cleanup.yml

```diff
@@ -0,0 +1,39 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Uninstall mirror-registry
+  become: true
+  cifmw.general.ci_script:
+    output_dir: "{{ cifmw_mirror_registry_basedir }}/artifacts"
+    script: |
+      set -o pipefail echo y | {{ cifw_mirror_registry_install_dir }}/mirror-registry uninstall -v
+  register: uninstall_output
```

Comment:
any reason to register the result?

Reply:
Na I just used it when I was testing it. I'll remove them both